configuration/index.md

```diff
@@ -128,14 +129,39 @@ Every interface has it's own strong points.
   </tbody>
 </table>
 
-{: #text-vs-ui}
 ### Textual vs. Graphical Configuration
 
-With openHAB 1.x the smart home configuration was done via configuration files only.
-One of the most visible additions to openHAB 2 is the administrative web interface Paper UI.
+In openHAB 1.X, smart home configuration was done via configuration files only.
+openHAB 2 added the administrative web interface "Paper UI". It allows for configuration of the base system and setup of openHAB 2
+addons. Please be aware that not every addon/binding that has been existing since openHAB version 1 has been updated to version 2 and
+thus to work with PaperUI. You can still run these but need to use file configuration for them.
 
 Things and Items can either be defined and managed in configuration files or handled by Paper UI in a [system-side database]({{base}}/administration/jsondb.html).
-
-Both methods can be used **side-by-side or even mixed**, e.g. a Thing is discovered by Paper UI and linked Items are defined in a `.items` configuration file.
-
-Configuration done in files will be synced to the database but configuration done in the database will not be written to files.
+Both methods can be used **side-by-side or even mixed**, e.g. a Thing is discovered by Paper UI and linked Items are defined in a 
+`.items` configuration file. Configuration done in files will be synced to the database (so it's accessible in PaperUI) but
```

Comment:
- add line break after "file."
- "Configuration done in files will be synced to the database" - this imho is not correct.
- it's -> they are
- PaperUI -> Paper UI
- remove line break at the end